src/NuGet/Microsoft.NETCore.Compilers.nuspec

```diff
@@ -26,13 +26,9 @@
     <file src="$additionalFilesPath$/Microsoft.NETCore.Compilers.props" target="build" />
 
     <!-- Targets and task files -->
-    <file src="Dlls/MSBuildTask/netstandard1.3/publish/Microsoft.Build.Tasks.CodeAnalysis.dll" target="tools" />
-    <file src="Dlls/MSbuildTask/netstandard1.3/publish/Microsoft.CSharp.Core.targets" target="tools" />
-    <file src="Dlls/MSbuildTask/netstandard1.3/publish/Microsoft.VisualBasic.Core.targets" target="tools" />
-    <!-- N.B.: The backslashes below cannot be replaced with forward slashes. 
-         https://github.com/NuGet/Home/issues/3584 -->
-    <file src="Dlls\MSBuildTask\netstandard1.3\publish\System.*.dll" target="tools" />
-    <file src="Dlls\MSBuildTask\netstandard1.3\publish\runtimes\**" target="tools\runtimes" />
```

Comment:
@nguerrera, @agocke please make sure to verify the last two `<file>` entries I deleted here is the right decision.

Reply:
Doesn't look right. You still need to carry the dlls that are not in netcoreapp2. What's left in the netcoreapp 2 publish folder?

Reply:
I expect pipes.accesscontrol.

Reply:
I was just including the output of build into the NuGet. Didn't think publish was necessary.

Reply:
Nope, publish is necessary.